lib/tty/prompt/list.rb

```diff
@@ -163,19 +171,45 @@ def arrows_help
         arrows.join
       end
 
+      # Convert a key name into a human-readable label
+      #
+      # @return [String]
+      #
+      # @api private
+      def key_help_label(key_name)
+        key_name == :return ? "Enter" : key_name.to_s.capitalize
+      end
```

Comment:
I think it's not too hard to add this 👍 

The display of keys in the hint is something I'm not totally happy about in this PR. I was wondering if there is a mapping that I missed between key symbols and actual keys in one of the existing dependencies, that could be used for the default display? It would be also nice for the dev to have some sort of reference to what key symbols are available/what they correspond to...

Reply:
There is a section which probably could do with making it more prominent. It references available keys [keyboard-events](https://github.com/piotrmurach/tty-prompt#212-keyboard-events). I guess we could link to it when advising about custom keys. As a side note, the keys handling comes from [tty-reader](https://github.com/piotrmurach/tty-reader).